loader/src/main/java/net/neoforged/fml/loading/FMLConfig.java

```diff
@@ -41,6 +41,7 @@ public enum ConfigValue {
         DEFAULT_CONFIG_PATH("defaultConfigPath", "defaultconfigs", "Default config path for servers"),
         DISABLE_OPTIMIZED_DFU("disableOptimizedDFU", Boolean.TRUE, "Disables Optimized DFU client-side - already disabled on servers"),
         EARLY_WINDOW_PROVIDER("earlyWindowProvider", "fmlearlywindow", "Early window provider"),
+        EARLY_WINDOW_GLDEBUG("earlyWindowGlDebug", Boolean.FALSE, "Enable OpenGL debugging in early window provider"),
```

Comment:
I think it would be a good idea to make this setting more generic and wire it into `RenderSystem.initRenderer()` in NF (for sync debug output the config would be passed directly to `initRenderer()` and for debug labels the config would be ORed together with vanilla's `renderDebugLabels` command line option). That way GL debugging controlled by this setting would be consistent across the entire game instead of some parts being enabled in one place but not another and, specifically for sync debug output, would also allow this setting to impact the game even if the early window is disabled.